test/sqlsmith/mzcompose.py

```diff
@@ -108,6 +108,9 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
             CREATE MATERIALIZED VIEW mv AS SELECT a + b AS col1, c + d AS col2, e + f AS col3, g AS col4 FROM t;
             CREATE MATERIALIZED VIEW mv2 AS SELECT count(*) FROM mv;
             CREATE DEFAULT INDEX ON mv;
+
+            CREATE MATERIALIZED VIEW mv3 WITH (REFRESH EVERY '2 seconds') AS SELECT * FROM counter;
+            CREATE DEFAULT INDEX ON mv3;
```

Comment:
Note that unfortunately we currently have a known bug when there is an index on a REFRESH mat view, see in the "Temporary Limitations" section of the [PR](https://github.com/MaterializeInc/materialize/pull/23819) description. (I'll open an issue for this tomorrow.) I think this shouldn't be a problem for this particular test as far as I can see, because 1) we are not doing restarts, 2) even after a restart, queries would block only for the refresh interval in the worst case, which is 2 seconds here. So I just wanted to mention this for other tests that you might create in the future.